huaweicloud/utils/utils.go

```diff
@@ -209,7 +209,7 @@ func EncodeBase64IfNot(str string) string {
 
 // IsIPv4Address is used to check whether the addr string is IPv4 format
 func IsIPv4Address(addr string) bool {
-	pattern := "^(25[0-5]|2[0-4]\\d|(1\\d{2}|[1-9]?\\d)\\.){3}(25[0-5]|2[0-4]\\d|(1\\d{2}|[1-9]?\\d))$"
+	pattern := "^((25[0-5]|2[0-4]\\d|(1\\d{2}|[1-9]?\\d))\\.){3}(25[0-5]|2[0-4]\\d|(1\\d{2}|[1-9]?\\d))$"
```

Comment:
please use the validation.IsIPv4Address instead

Reply:
the return values of validation.IsIPv4Address is not suitable

Reply:
Whether the regular expression for this method should be represented as three types of IPv4 addresses: A, B and C.
A: `1.0.0.1` to `127.255.255.254`
B: `128.0.0.1` to `191.255.255.254`
C: `192.0.0.1` to `223.255.255.254`

Reply:
the function is apply for all IPv4 addresses: 0.0.0.0 - 255.255.255.255